scalalib/src/CrossSbtModule.scala

```diff
@@ -1,24 +1,29 @@
 package mill.scalalib
 
+import mill.api.PathRef
 import mill.T
 import mill.scalalib.{CrossModuleBase, SbtModule}
 
 trait CrossSbtModule extends SbtModule with CrossModuleBase { outer =>
 
   override def sources = T.sources {
+    val crossScalaVersions = millOuterCtx.crossInstances.map(_.asInstanceOf[CrossScalaModule].crossScalaVersion)
```

Comment:
Here I would love to maintain the type of the `crossInstances` without them being of type `AnyRef`, but I couldn't find a way to achieve that.